frontend/src/components/custom-tools/prompt-card/PromptOutputsModal.jsx

```diff
@@ -78,7 +79,9 @@ function PromptOutputsModal({
                     )}
                   </div>
                   <div className="flex-1 overflow-y-auto pad-top-10">
-                    {enforceType === TABLE_ENFORCE_TYPE && TableOutput ? (
+                    {(enforceType === TABLE_ENFORCE_TYPE ||
+                      enforceType === RECORD_ENFORCE_TYPE) &&
```

Comment:
@jagadeeswaran-zipstack NIT: Can we create a variable called `isTableExtraction` for the above condition here? Similar to the one in `PromptOutput.jsx` component. A better approach here would be to pass the `isTableExtraction` prop from `PromptOutput` to `PromptOutputsModal`. Please check if this is possible.

Reply:
I Don't think its needed here. Because We're using this in only one place. So it will be overkill here. Also passing down `isTableExtraction` as prop is not a better way to do this because in future we might need other enforce type other than table and record.